.circleci/config.yml

```diff
@@ -86,7 +86,7 @@ jobs:
           name: check documentation spelling errors
           command: 'codespell -I ./words_to_ignore.txt docs/'
 
-  gometalinter:
+  lint:
```

Comment:
golint? or backend-lint?

Reply:
`backend-lint` sounds good to me! Since we wouldn't use `golint` :)